### DISCUSSION TITLE
blog/flinks/index

### DISCUSSION THREAD

**Discussion by giscus** [sumy7/sumy7.github.io#13] [Announcements]
# blog/flinks/index

SumyGG's Blog

https://sumygg.com/blog/flinks/index.html

**Comment by citrusjunoss** [comment]
大佬你好，新建的博客使用了你的主题。感谢🙏，方便的话可以交换个友链么，[乌漆嘛黑](https://www.chengzila.com)

**Reply by sumy7** [reply]
感谢使用主题。友链已添加。（迟到了一年。。。

**Comment by SM202110** [comment]
你好，我想与你交换链接
starriver.xin
如果准备好了，请用邮件联系我：smwork@s-m-e-l.us.kg

**Comment by code-ooo** [comment]
大佬，这是我的博客，请问你的友链，如何添加：{
  "title": "口袋分享记",
  "link": "https://111620.xyz/",
  "logo": "https://img.111620.xyz/2025/07/29/6887b5efbf768.png",
  "desc": "所谓过往，皆为序章。虚室生白️",
}

**Reply by sumy7** [reply]
感谢来访。。。友链已添加。。。我的可以按照以下格式： {
"title": "SumyBlog",
"link": "https://sumygg.com/",
"logo": "https://sumygg.com/favicon.ico",
"desc": "",
}

**Reply by code-ooo** [reply]
🆗，大佬，添加上了！
